server/routes/products.js

```diff
@@ -1,17 +1,34 @@
 const express = require('express');
 const products = express.Router();
+const API = require('../../apiHelpers/atelier')
 
 // retrieves the list of products. takes in 2 parameters {page, count}
 products
   .route('/')
   .get((req, res) => {
-    res.status(200).send('hello from products!')
+    API.getProducts((err, response) => {
+      if (err) {
+        console.log(err)
+        res.status(400).send("error from products:", err)
+      } else {
+        res.status(200).send(response)
```

Comment:
If we want to handle errors here, it should look like

.get((req, res) => {
    apiHelpers.getProductList()
      .then(data => {
        res.status(200).send(data.data)
      })
      .catch(err => {
        res.status(404).send(err);
      })
  })

no need to use error first callbacks when using promise based syntax

Reply:
Alternatively, this can be as simple as 

.get(API.getProducts)

and the error handling and data sending can happen on Atelier.js. this is preference

Reply:
I think for now it will be helpful to have error handling on both ends... at least until our test suite is operational :)